lib/Objects/active_time_ranges.dart

```diff
@@ -0,0 +1,25 @@
+/// Copyright 2020 Google LLC
```

Comment:
lib/Objects/active_time_ranges.dart  should be lib/objects/active_time_ranges.dart

Reply:
Done.